src/azure-cli/azure/cli/command_modules/appservice/_create_util.py

```diff
@@ -203,8 +214,11 @@ def parse_node_version(file_path):
                 # remove the string ~ or  > that sometimes exists in version value
                 c = non_decimal.sub('', value_detected)
                 # reduce the version to '6.0' from '6.0.0'
-                num_array = c.split('.')
-                num = num_array[0] + "." + num_array[1]
+                if '.' in c:  # handle version set as 4 instead of 4.0
+                    num_array = c.split('.')
+                    num = num_array[0] + "." + num_array[1]
```

Comment:
so, this will also get rid of the build and revision version and formalize it as `<major>.<minor>`, right?

Reply:
yes that's what we always do here since we pass this as the runtime version to the create, all i am changing here is if we get just major version we change it major.minor